config/webpack.prod.config.js

```diff
@@ -41,6 +41,10 @@ const config = {
       }
     ]
   },
+  optimization: {
+    // Minimize and uglify the code
+    minimizer: [new TerserPlugin()]
```

Comment:
so it doesn't work at all without specifying this, right? cause technically `minimize: true` is meant to just use the [TerserPlugin anyways](https://webpack.js.org/configuration/optimization/#optimizationminimize)

Reply:
ohhhh ! I see you're using `minimizer` _not_ `minimize`! nvm then

Reply:
with `minimize: true` it doesn't work for me. i have a problem with removing npm modules because of `terser-webpack-plugin` cash being locked.

Reply:
At some point in time I'd be curious if a brand new plugin will also not work for you, or if it's just with this particular plugin's setup. Cause I feel like cache being locked is probably a machine issue, not sure tho!

Reply:
i have created a new plugin to check and have the same issue:
```
Alenas-MBP:test2 alenakhineika$ rm -r node_modules
override r--r--r--  alenakhineika/staff for node_modules/.cache/terser-webpack-plugin/content-v2/sha512/9d/f2/65f8a8f30c48f4d2c1c4a3294c4a79779986422fd732ac84bf68d905fffa596d60937dc6c604efcd685e45ade43f9b885dce60f2523dc56ab90c094adb08?
```

Reply:
ah! you probably gotta force it `rm -rf`; node_modules are _so_ finicky :sob: